components/common/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from "react";
+import Link from "next/link";
+import Image from "next/image";
+import styles from "./navbar.module.css";
+import Profile from "./Profile/Profile";
+import { fetchGetSampleUsers } from "hooks/useGetSampleData";
+import { ID, USER_ID } from "constants/strings";
+import type { User } from "types";
+
+function Navbar() {
+  const [user, setUser] = useState<User>();
+
+  const setLocalStorage = (key: string, val: number) => {
+    localStorage.setItem(key, JSON.stringify(val));
+  };
```

Comment:
컴포넌트 외부에 있어도 되는 함수로 보이네요!

Reply:
이렇게 표현해주는게 더 좋을 것 같아요~
```suggestion
  const setLocalStorage = (key: string, value: number) => {
    localStorage.setItem(key, JSON.stringify(value));
  };
```

Reply:
로컬 스토리지를 어떻게 모듈화해서 사용할지 고민해보면 좋은데요~
저는 매번 사용할때마다 key값을 넣어줘야 하는게 조금 아쉬운 것 같아요.
value값만 넣어주는 방식으로 사용하는건 어떨까요?